assets/css/formbuilder.css

```
@@ -486,6 +515,18 @@ body.rex-theme-dark .mform-fb {
     color: var(--fb-text-faint);
 }
 
+.mform-fb__colorswatch-help {
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)